src/BKT/Bai2/Student.java

```diff
@@ -0,0 +1,66 @@
+package BKT.Bai2;
+
+import BKT.Bai2.Address;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Student extends Person{
+    static Scanner scanner = new Scanner(System.in);
+    private int id;
+    private Address address = new Address();
+    private double gpa;
+
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public Address getAddress() {
+        return address;
+    }
+
+    public void setAddress(Address address) {
+        this.address = address;
+    }
+
+    public double getGpa() {
+        return gpa;
+    }
+
+    public void setGpa(double gpa) {
+        this.gpa = gpa;
+    }
+
+    public Student(){}
+
+    public Student(String name, int age, String gender, int id, Address address, double gpa) {
+        super(name, age, gender);
+        this.id = id;
+        this.address = address;
+        this.gpa = gpa;
+    }
+    @Override
+    public void input()
+    {
+        System.out.print("Nhập id: ");
+        id = scanner.nextInt();
+        super.input();
+        address.input();
+        System.out.print("Nhập 0 <= gpa <= 4: ");
```

Comment:
Tiêu đề in ra như vậy nhưng người dùng không thích nhập trong khoảng thì sao nhẻ? (-5đ)

Reply:
Ớ đề bài có yêu cầu bắt buộc nhập đâu